pages_builder/pages/forms/selection-buttons.yml

```diff
@@ -252,7 +252,9 @@ examples:
     question: Can you do this essential requirement
     question_number: 93
     value: true
-    followup: requirement-evidence-110
+    followup:
+      true:
+        - requirement-evidence-110
```

Comment:
This is not YAML format used by the content loader, it's the data we'd normally pass into toolkit form macros, so this is what `.values_followup` would return, which is a reversed mapping.